framework/wazuh/core/config/models/logging.py

```diff
@@ -0,0 +1,90 @@
+import logging
+
+from pydantic import BaseModel, Field
+from typing import Literal, List
+
+
+class LoggingConfig(BaseModel):
+    """Configuration for logging levels.
+
+    Parameters
+    ----------
+    level : Literal["info", "debug", "debug2"]
+        The logging level. Default is "info".
+    """
+    level: Literal["info", "debug", "debug2"] = "info"
+
+    def get_level_value(self) -> int:
+        """Returns the integer value associated with the logging level.
+
+        Returns
+        -------
+        int
+            The integer value corresponding to the logging level:
+            - 0 for "info"
+            - 1 for "debug"
+            - 2 for "debug2"
+        """
+        if self.level == "info":
+            return 0
+        elif self.level == "debug":
+            return 1
+        else:
+            return 2
```

Comment:
Similarly, we could use a dictionary of enumerator -> value

```suggestion
        if self.level == "info":
            return 0
        if self.level == "debug":
            return 1

        return 2
```

Reply:
Changed in e4e54327c2b3a45cbe77a6f9c30cfd19dc9ce0b6.